.storybook/addons/usage/register.js

```diff
@@ -0,0 +1,184 @@
+import { addons, types } from '@storybook/addons';
+import { useParameter } from '@storybook/api';
+import { AddonPanel } from '@storybook/components';
+import 'github-markdown-css/github-markdown.css';
+import React from 'react';
+
+const ADDON_ID = 'axa-ch/usage';
+const PANEL_ID = `${ADDON_ID}/panel`;
+
+const formatComponentName = componentName => {
+  if (componentName === undefined) {
+    return {};
+  }
+
+  const pureHTML = componentName.toLowerCase();
+  const formatReactTag = componentName => {
+    let result = componentName.substring(0, 1).toUpperCase();
+
+    for (let index = 1; index < componentName.length; index++) {
+      const char = componentName.charAt(index);
+
+      if (char === '-') {
+        index++;
+        result = result + componentName.charAt(index).toUpperCase();
+      } else {
+        result = result + char;
+      }
+    }
```

Comment:
If you want to become swagmaster2000, use this ternary one-liner instead of line 22-27:

```result = char === '=' ? result + componentName.charAt(++index).toUpperCase() : result + char;```